builtin/providers/aws/config.go

```diff
@@ -84,9 +86,15 @@ func (c *Config) Client() (interface{}, error) {
 			Region:      c.Region,
 			MaxRetries:  c.MaxRetries,
 		}
+		awsDynamoDBConfig := &aws.Config{
+			Credentials: creds,
+			Region:      c.Region,
+			MaxRetries:  c.MaxRetries,
+			Endpoint:    c.DynamoDBEndpoint,
+		}
```

Comment:
I think this is fine for now - the duplication is super local. If we get more duplication here we can investigate cleanup options. :ok_hand: